### PR TITLE
[mds-jurisdiction] Export needed jurisdiction types

### PR DESCRIPTION
**Pull request by twelch**
## 📚 Purpose
For API client support

## 📦 Impacts:
mds-jurisdiction

### CODE REVIEW

File: packages/mds-jurisdiction/handlers/create-jurisdiction.ts [hunk 1]
```diff
@@ -22,17 +22,23 @@ import {
...
 import { JurisdictionApiRequest, JurisdictionApiResponse } from '../@types'
 
+export type JurisdictionApiCreateJurisdictionRequestBody =
+  | CreateJurisdictionDomainModel
+  | CreateJurisdictionDomainModel[]
+
 export type JurisdictionApiCreateJurisdictionRequest = JurisdictionApiRequest<
-  CreateJurisdictionDomainModel | CreateJurisdictionDomainModel[]
+  JurisdictionApiCreateJurisdictionRequestBody
 >
```
Comment (drtyh2o) [inline packages/mds-jurisdiction/handlers/create-jurisdiction.ts:31]: You could minimize the number of changes by altering the way these types are declared. For instance:
```ts
export type JurisdictionApiCreateJurisdictionRequest = JurisdictionApiRequest<
  CreateJurisdictionDomainModel | CreateJurisdictionDomainModel[]
>

export type JurisdictionApiCreateJurisdictionRequestBody = JurisdictionApiCreateJurisdictionRequest['body']
```
Additionally, you _could_ just use `JurisdictionApiCreateJurisdictionRequest['body']` and not declare the additional type at all although I am certainly not opposed to having a separate named type.

**Review by drtyh2o — APPROVED** [review]
Small comment for your consideration.

File: packages/mds-jurisdiction/handlers/create-jurisdiction.ts (hunk 1 above)
Comment (twelch) [reply packages/mds-jurisdiction/handlers/create-jurisdiction.ts:31]: It's coming clearer to me.  On that note I think I can just use `JurisdictionApiCreateJurisdictionRequest['body']` on the client side and just need to export what we have already.  What took me down this path was seeing that geography split things up like this

Comment (twelch) [reply packages/mds-jurisdiction/handlers/create-jurisdiction.ts]: Okay I've got the request types trimmed down.  It took me a while to realize that the ApiResponse is a different animal and doesn't have a body in the same way and isn't used on the client-side in the same way.

**Comment by twelch** [comment]
This is ready for sign-off if I can get a 👍 @avatarneil or @BigIdeas

**Review by evanxadkins — APPROVED** [review]

**Review by avatarneil — APPROVED** [review]

**Pull request merged**